jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -58,6 +54,57 @@ fetch_k8s_logs()
     done
 }
 
+fetch_manifests()
+{
+    dir_name="${1}"
+    kconfig="$2"
+
+    manifests=(
+      bmh
+      hardwaredata
+      cluster
+      deployment
+      machine
+      machinedeployment
+      machinehealthchecks
+      machinesets
+      machinepools
+      m3cluster
+      m3machine
+      metal3machinetemplate
+      kubeadmconfig
+      kubeadmconfigtemplates
+      kubeadmcontrolplane
+      replicaset
+      ippool
+      ipclaim
+      ipaddress
+      m3data
+      m3dataclaim
+      m3datatemplate
+    )
+
+    NAMESPACES="$(kubectl --kubeconfig="${kconfig}" get namespace -o jsonpath='{.items[*].metadata.name}')"
+    for NAMESPACE in ${NAMESPACES}; do
+      for kind in "${manifests[@]}"; do
+        mkdir -p "${LOGS_DIR}/${dir_name}/${kind}"
+        for name in $(kubectl --kubeconfig="${kconfig}" get -n "${NAMESPACE}" -o name "${kind}" || true); do
+          kubectl --kubeconfig="${kconfig}" get -n "${NAMESPACE}" -o yaml "${name}" | tee "${LOGS_DIR}/${dir_name}/${kind}/$(basename "${name}").yaml" || true
```

Comment:
if we have the same kind in different namespace then would those get overridden ?

Reply:
Good catch, and we probably have the same issue in `fetch_manifests.sh` where this was copied from. Namespace must be part of the filename or path, or you need to fetch all at once, not per namespace.

/hold

Reply:
If I use the Namespace as part of the path, that should solve our issue, isn't it?

Reply:
You need to put the namespace to the `mkdir` as well.